test/core/test_vectors/test.cpp

```diff
@@ -256,11 +256,23 @@ auto search() {
           kCorpusRoot + "/extracted/0004-coverage-boost/fil_1_storageminer/SubmitWindowedPoSt/Ok/ext-0004-fil_1_storageminer-SubmitWindowedPoSt-Ok-9.json"
       };
 
+      // Skip tests that fail in Fuhon
+      // TODO (@wer1st) these tests should be enabled after FIL-260
```

Comment:
Please, add brief description of the problem, so developers haven't search for FIL-260 every time they see the TODO.

Reply:
done